chore: update contribution (#596)
jina-ai/docarray: CONTRIBUTING.md

pytest -v -s tests
```

### Test policy

> Every contribution that add or modify the behavior of a features must include a suit of tests that's validate the feature work as excepted 

Reviewer: grammar correction, let's wait for @NicholasDunham to review

> Every contribution that add or modify the behavior of a features must include a suit of tests that's validate the feature work as excepted 

Reviewer 2: ```suggestion
Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates the feature work as excepted.
```

> Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates the feature work as excepted.

Reviewer 3: ```suggestion
Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates that the feature work as excepted.
```

This allows:

> * reviewer to be very confident that the feature does what it is supposed to do before merging it into the codebase

Reviewer 2: ```suggestion
* the reviewer to be very confident that the feature does what it is supposed to do before merging it into the code base
```

> * contributors to be sure that they don't break already merged features when doing refactoring/modification on the codebase.

Reviewer 2: ```suggestion
* the contributors to be sure that they don't break already merged features when doing refactoring/modification on the code base.
```

> * the contributors to be sure that they don't break already merged features when doing refactoring/modification on the code base.

Reviewer 4: ```suggestion
* the contributors to be sure that they don't break already-merged features when refactoring or modifying the code base.
```
